crates/relayer/src/chain/axon.rs

```diff
@@ -308,22 +325,25 @@ impl ChainEndpoint for AxonChain {
         self.light_client.check_misbehaviour(update, client_state)
     }
 
-    // FIXME implement this after use a real ics token contract
-    fn query_balance(
-        &self,
-        _key_name: Option<&str>,
-        _denom: Option<&str>,
-    ) -> Result<Balance, Error> {
-        // const DEFAULT_DENOM: &str = "AT";
-
-        // let key_name = key_name.unwrap_or(&self.config.key_name);
-        // let denom = denom.unwrap_or(DEFAULT_DENOM);
-        // let contract = self.ics20_contract()?;
-        // let wallet =  self.get_wallet(key_name);
+    fn query_balance(&self, key_name: Option<&str>, denom: Option<&str>) -> Result<Balance, Error> {
+        let key_name = key_name.unwrap_or(&self.config.key_name);
+        let denom: &str =
+            denom.ok_or_else(|| Error::other_error("do not support default denom".into()))?;
+        let erc20_address = {
```

Comment:
use `denom` as erc20 address is compatible with the previous use in CI?

<img width="435" alt="image" src="https://github.com/synapseweb3/forcerelay/assets/3871265/f8d82442-cabd-4f8a-a6cb-da8affd236cf">

Reply:
Yes, by my understanding, the CI is using erc20 address as the env `DENOM`.